src/components/WebChat/WebChat.css

```diff
@@ -1,117 +1,112 @@
-.webchat-container{
```

Comment:
This file is no longer being used so can be removed.

Reply:
File removed, thanks